sdk/servicebus/azure-servicebus/tests/perf_tests/receive_topic_message_batch.py

```diff
@@ -5,27 +5,27 @@
 
 import asyncio
 
-from ._test_base import _ReceiveTest
+from ._test_base import _TopicReceiveTest
 
 
-class ReceiveMessageBatchTest(_ReceiveTest):
-    def run_sync(self):
+class ReceiveTopicMessageBatchTest(_TopicReceiveTest):
```

Comment:
I think all the receive topics should be ReceiveSubscription, since we can only send to topics.